test/srt/test_fused_moe.py

```diff
@@ -98,22 +97,11 @@ def _test_case(self, m, n, k, e, topk, dtype, use_fp8_w8a8=False):
                 a2_scale=a2_scale,
             )
 
-            vllm_output = fused_moe_vllm(
-                a,
-                w1,
-                w2,
-                score,
-                topk,
-                renormalize=False,
-                use_fp8_w8a8=True,
-                w1_scale=w1_scale,
-                w2_scale=w2_scale,
-                a1_scale=a1_scale,
-                a2_scale=a2_scale,
+            torch_output = self.torch_naive_moe(a, w1, w2, score, topk)
```

Comment:
nit: a bit confused, it seems we need the w1_scale etc when doing moe

Reply:
It's no necessary, please refer to https://github.com/deepseek-ai/DeepGEMM/blob/main/tests/test_core.py#L33 .

Reply:
Fixed.